src/librawspeed/tiff/TiffEntry.h

```diff
@@ -78,7 +78,7 @@ class TiffEntry
   ushort16 getShort(uint32 num=0) const;
   short16 getSShort(uint32 num=0) const;
   float getFloat(uint32 num=0) const;
-  std::string getString() const;
+  std::string getString(bool trimmed = true) const;
```

Comment:
Would you please explain to me what the danger is exactly?

Please note:
* rstest is happy
* if someone wrote some new code and called this function and trimming was not what he wanted, then he can call the function an pass 'false'.
* based on the code it is more likely that some new code actually needs yet another trim call and the developer forgets about it until he notices it later.

Alternative suggestion, which makes it more obvious that trimming is happening: have two functions, one called `getTrimmedString()` and use that. We'd get about the same effect, just more characters.